h/static/scripts/permissions.coffee

```diff
@@ -37,10 +37,15 @@ module.exports = ['session', (session) ->
   # Typical use: annotation.permissions = permissions.private()
   ###
   private: ->
-    read: [session.state.userid]
-    update: [session.state.userid]
-    delete: [session.state.userid]
-    admin: [session.state.userid]
+    if not session?.state?.userid
```

Comment:
Yeah, I know the existing tests will pass without this.

One thing that I'm not happy with about our client-side tests is the amount of detail faking needed, though. The AnnotationController tests need a fake session service with a state, and a fake permissions serve with several methods, etc etc. If we make a change to the interface of any of these services then all our test fakes become out of date.